RELEASE_PROCESS.md

```diff
@@ -0,0 +1,186 @@
+# Release Process
+
+- [Release Process](#release-process)
+  - [Major Release Procedure](#major-release-procedure)
+    - [Changelog](#changelog)
+    - [Release Notes](#release-notes)
+    - [Tagging Procedure](#tagging-procedure)
+      - [Test building artifacts](#test-building-artifacts)
+      - [Installing goreleaser](#installing-goreleaser)
+  - [Non-major Release Procedure](#non-major-release-procedure)
+  - [Major Release Maintenance](#major-release-maintenance)
+  - [Stable Release Policy](#stable-release-policy)
+
+
+This document outlines the release process for Cosmos Hub (Gaia).
+
+Gaia follows [semantic versioning](https://semver.org), but with the following deviations to account for state-machine and API breaking changes: 
+
+- State-machine breaking changes will result in an increase of the major version X (X.y.z).
+- API breaking changes will result in an increase of the minor version Y (x.Y.z | x > 0).
+- All other changes will result in an increase of the patch version Z (x.y.Z | x > 0).
+
+**State compatibility**: 
+It is critical for the patch and minor releases to be state-machine compatible with prior releases in the same major version. 
```

Comment:
This should be applicable to patch releases. Not sure minor releases should maintain state compatibility, since they only arise from emergency situations